### ISSUE TITLE
Support and ovrhanging solution

### ISSUE THREAD

**Issue by misiek271** [supermerill/SuperSlicer#3412]
**Is your feature request related to a problem? Please describe.**
Hi we have big problem with support adhesion
to much support interface adhesion. like on photo. When you print your object on support Ss tells that it is a overhang and bridge. That's is very bad.Because overhanging is printing with minimal speed for ex. 10mm/s. when you print real overhang with no support with that speed its, ok but when you print on a support with that speed plastic adhesion to support is incredible strong so you can't remove support. 

**Describe the solution you'd like**
make different name for First permiter above support interface and first infil above support interface and give us setting to change its speed. and fan cooling. this setting can give us powerfull tool too good looking and easy removing supports, and real overhanging(with no support) will be good too because they have own speed value. 

basically make 2 new names for infil above interface support and permiter above support and give them way to change speed and cooling

**Describe alternatives you've considered**
alternative solutions that i try was changing speed for overhang like 40mm/s but in that case if you have support and real overhang with no support in one object you must chose between support adhesion to object or good looking overhang. so that isn;t solution

![overr]
![speed](https://user-images.githubusercontent.com/71993614/200437135-2e1a1733-886d-4dca-a189-968e4f83cc46.jpg)
(https://user-images.githubusercontent.com/71993614/200437127-470f3b85-6052-430d-8789-1185e3ba07ed.jpg)
![supp](https://user-images.githubusercontent.com/71993614/200437154-defa1977-7a38-4a0d-83f9-07b877ebc5dc.jpg)

**Comment by misiek271** [comment]
Isory i atteched wrong file

![overr](https://user-images.githubusercontent.com/71993614/200437270-e60e7cec-03b9-4863-a63d-ef13192ca7e0.jpg)
